elastic/beats
# Add host inventory metrics to azure compute_vm metricset

## What does this PR do?

This PR is to add proposed host common fields into `compute` metricset:
- host.id
- host.name
- host.cpu.pct
- host.network.in.bytes
- host.network.in.packets
- host.network.out.bytes
- host.network.out.packets
- host.disk.read.bytes
- host.disk.write.bytes

## Why is it important?



This PR is a part of the inventory schema work. 

## Checklist




## How to test this PR locally
1. Start Metricbeat `azure` module: `./metricbeat modules enable azure`
2. Edit `modules.d/azure.yml` to only include `compute_vm` metricset:
```
- module: azure
  metricsets:
    - compute_vm
  enabled: true
  period: 300s
  client_id: '123'
  client_secret: '123'
  tenant_id: '123'
  subscription_id: '123'
  refresh_list_interval: 600s
```
3. Change `add_host_metadata` processor config in metricbeat.yml file:
```
processors:
  - add_host_metadata:
      replace_fields: false
```
4. Start metricbeat
5. You should see metrics from `compute_vm` metricset and includes fields listed above.
